docs/source/user_guide/cuspatial_api_examples.ipynb

```diff
@@ -166,8 +166,9 @@
      "name": "stderr",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.    print(df)
</pre><p>Can we use df.head() instead to reduce the output size?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cuspatial/pull/1259/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>